### PR TITLE
@orta => Updated lockout

### LINKED ISSUE #146: Make sure partners who should be locked out are locked out
Look into logged-in partners who have already synced.

### PR DESCRIPTION
**Pull request by sarahscott**
This is a quick fix for locking out ineligible partners if they refuse to sync again, but I'm making a separate issue to make lockout more robust in general and easier to use as an API.

Fixes #146

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Error
    ------------- | ------------
        :no_entry_sign: | No CHANGELOG changes made
    

  

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

File: Classes/ARAppDelegate.m [hunk 1]
```diff
@@ -119,15 +118,18 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
         [ARAnalytics identifyUserWithID:[User currentUser].slug andEmailAddress:[User currentUser].email];
 
-        [_partnerSync performPartnerMetadataSync:^{
-            if (![Partner currentPartner].hasUploadedWorks) {
+        [self updatePartnerMetadataCompletion:^{
+            if (self.shouldLockout) {
+                [ARAnalytics event:ARLockoutEvent];
+                [self.viewCoordinator presentLockoutScreenContext:[CoreDataManager mainManagedObjectContext]];
+            } else if (![Partner currentPartner].hasUploadedWorks) {
```
Comment (orta) [inline Classes/ARAppDelegate.m]: can you add some logical comments, this app launch stuff is critical

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by sarahscott** [comment]
:green_apple:

**Comment by orta** [comment]
:+1:

**Pull request merged**